src/index.tsx

```diff
@@ -1,10 +1,11 @@
 import { createRoot } from 'react-dom/client';
 
+import { HashRouter } from 'react-router-dom';
```

Comment:
It looks like you added react-router-dom's HashRouter to the application, but the task does not require implementing routing. Please remove the HashRouter import and its usage in the render function.